Dragon does not appear

### Description
#### Describe the bug
<!-- A clear and concise description of the problem you're encountering. -->
<!-- /!\ Leaving this section blank will result in your ticket being closed without further explanation. -->
<!-- Please type below this line. -->
After all, towers spawn, the dragon does not spawn and in console spam errors.

#### Steps to reproduce the behavior
<!-- Step-by-step instructions for us to reproduce the bug on our side. -->
<!-- /!\ Leaving this section blank will result in your ticket being closed without further explanation. -->
<!-- Please type below this line. -->
Put crystals to spawn dragon and wait.

#### Expected behavior
<!-- What do you think should happen? -->
<!-- Please type below this line. -->
Dragons appears and no errors in console.

### Environment

##### BentoBox Version (Mandatory)
<!-- /!\ Leaving this section blank will result in your ticket being closed without further explanation. -->
<!-- Paste the output of `/bentobox version` in the code block below (replace the underscores). Do not provide an image. -->
```
[10:47:26 INFO]: Running PAPER Netinkama.
[10:47:26 INFO]: BentoBox versija: 1.21.0-SNAPSHOT-b2283
[10:47:26 INFO]: Database: MARIADB
[10:47:26 INFO]: Loaded Game Worlds:
[10:47:26 INFO]: oneblock_world (OneBlock): Paviršiaus pasaulis, Neteris, Pabaiga
[10:47:26 INFO]: Loaded Addons:
[10:47:26 INFO]: AOneBlock 1.11.0-SNAPSHOT-b268 (ENABLED)
[10:47:26 INFO]: Bank 1.4.0-SNAPSHOT-b62 (ENABLED)
[10:47:26 INFO]: Biomes 2.1.0-SNAPSHOT-b230 (ENABLED)
[10:47:26 INFO]: Border 4.0.0-SNAPSHOT-b144 (ENABLED)
[10:47:26 INFO]: Challenges 1.1.0-SNAPSHOT-b490 (ENABLED)
[10:47:26 INFO]: DimensionalTrees 1.6.1-SNAPSHOT-b84 (ENABLED)
[10:47:26 INFO]: ExtraMobs 1.12 (ENABLED)
[10:47:26 INFO]: Greenhouses 1.7.0 (ENABLED)
[10:47:26 INFO]: Level 2.9.1-SNAPSHOT-b510 (ENABLED)
[10:47:26 INFO]: Likes 2.4.0-SNAPSHOT-b89 (ENABLED)
[10:47:26 INFO]: Limits 1.19.1-SNAPSHOT-b300 (ENABLED)
[10:47:26 INFO]: MagicCobblestoneGenerator 2.5.0-SNAPSHOT-b251 (ENABLED)
[10:47:26 INFO]: TwerkingForTrees 1.4.4-SNAPSHOT-b59 (ENABLED)
[10:47:26 INFO]: Warps 1.13.0-SNAPSHOT-b328 (ENABLED)
DragonFights-1.4.2
```

#### Plugins (Optional)
<!-- Please paste the `/plugins` output inside the code block below (remove the underscores). Do not provide an image. -->
```
AdvancedRegionMarket, BentoBox, BungeeTabListPlus, ChestShop, Chunky, Citizens, ColoredAnvils*, CoreProtect, DeathMessages, DisableFriendlyFire, EntityLimiter, EZRanksLite*, FastAsyncWorldEdit (WorldEdit), FriendsAPIForPartyAndFriends, GHolo, GSit, Hat, HungerKeeperPlus*, InventoryGUI, JetsHoppers, JetsMinions, JoinDelay, KCore, KillerMoney*, Kits, LibsDisguises, Localization, LunarClient-API*, LunarClient-Impl*, Market, MarriageMaster, mcMMO, Multiverse-Core, Permissions, PlayerHeads, PlayerVaults, PlugMan, Portals, ProtocolLib, ProxyBridge, SellStick, SignEdit, SkinsRestorer, spark, Spartan, SpawnerShop, Trading, UltraCosmetics, Vault, VoidGen, WorldBorderAPI, WorldGuard
```
 
### Additional context (Optional)
<!-- Any additional information you'd like to provide us. -->
<!-- Please type below this line. -->
`[Server thread/WARN]: [BentoBox] Task #460 for BentoBox v1.21.0-SNAPSHOT-b2283 generated an exception
java.lang.RuntimeException: java.lang.reflect.InvocationTargetException
	at lv.id.bonne.dragonfights.v1_19_R1.NMSEntityRegistry.lambda$applyRegister$0(NMSEntityRegistry.java:136) ~[DragonFights-1.4.2.jar:?]
	at net.minecraft.world.entity.EntityType.create(EntityType.java:512) ~[?:?]
	at lv.id.bonne.dragonfights.v1_19_R1.battle.BentoBoxDragonBattle.lambda$createNewDragon$8(BentoBoxDragonBattle.java:1312) ~[DragonFights-1.4.2.jar:?]
	at java.util.Optional.map(Optional.java:260) ~[?:?]
	at lv.id.bonne.dragonfights.v1_19_R1.battle.BentoBoxDragonBattle.createNewDragon(BentoBoxDragonBattle.java:1312) ~[DragonFights-1.4.2.jar:?]
	at lv.id.bonne.dragonfights.v1_19_R1.battle.BentoBoxDragonBattle.findOrCreateDragon(BentoBoxDragonBattle.java:1272) ~[DragonFights-1.4.2.jar:?]
	at lv.id.bonne.dragonfights.v1_19_R1.battle.BentoBoxDragonBattle.tickBattle(BentoBoxDragonBattle.java:417) ~[DragonFights-1.4.2.jar:?]
	at lv.id.bonne.dragonfights.managers.DragonFightManager$BattleTick.accept(DragonFightManager.java:457) ~[DragonFights-1.4.2.jar:?]
	at lv.id.bonne.dragonfights.managers.DragonFightManager$BattleTick.accept(DragonFightManager.java:405) ~[DragonFights-1.4.2.jar:?]
	at org.bukkit.craftbukkit.v1_19_R1.scheduler.CraftTask.run(CraftTask.java:103) ~[paper-1.19.2.jar:git-Paper-176]
	at org.bukkit.craftbukkit.v1_19_R1.scheduler.CraftScheduler.mainThreadHeartbeat(CraftScheduler.java:483) ~[paper-1.19.2.jar:git-Paper-176]
	at net.minecraft.server.MinecraftServer.tickChildren(MinecraftServer.java:1473) ~[paper-1.19.2.jar:git-Paper-176]
	at net.minecraft.server.dedicated.DedicatedServer.tickChildren(DedicatedServer.java:446) ~[paper-1.19.2.jar:git-Paper-176]
	at net.minecraft.server.MinecraftServer.tickServer(MinecraftServer.java:1397) ~[paper-1.19.2.jar:git-Paper-176]
	at net.minecraft.server.MinecraftServer.runServer(MinecraftServer.java:1173) ~[paper-1.19.2.jar:git-Paper-176]
	at net.minecraft.server.MinecraftServer.lambda$spin$0(MinecraftServer.java:305) ~[paper-1.19.2.jar:git-Paper-176]
	at java.lang.Thread.run(Thread.java:833) ~[?:?]
Caused by: java.lang.reflect.InvocationTargetException
	at jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method) ~[?:?]
	at jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:77) ~[?:?]
	at jdk.internal.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45) ~[?:?]
	at java.lang.reflect.Constructor.newInstanceWithCaller(Constructor.java:499) ~[?:?]
	at java.lang.reflect.Constructor.newInstance(Constructor.java:480) ~[?:?]
	at lv.id.bonne.dragonfights.v1_19_R1.NMSEntityRegistry.lambda$applyRegister$0(NMSEntityRegistry.java:132) ~[DragonFights-1.4.2.jar:?]
	... 16 more
Caused by: java.lang.NoSuchMethodError: 'void lv.id.bonne.dragonfights.v1_19_R1.entity.BentoBoxEnderDragon.fq()'
	at lv.id.bonne.dragonfights.v1_19_R1.entity.BentoBoxEnderDragon.<init>(BentoBoxEnderDragon.java:70) ~[DragonFights-1.4.2.jar:?]
	at jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method) ~[?:?]
	at jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:77) ~[?:?]
	at jdk.internal.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45) ~[?:?]
	at java.lang.reflect.Constructor.newInstanceWithCaller(Constructor.java:499) ~[?:?]
	at java.lang.reflect.Constructor.newInstance(Constructor.java:480) ~[?:?]
	at lv.id.bonne.dragonfights.v1_19_R1.NMSEntityRegistry.lambda$applyRegister$0(NMSEntityRegistry.java:132) ~[DragonFights-1.4.2.jar:?]
	... 16 more`

--- a/pom.xml
+++ b/pom.xml
@@ -26,7 +26,7 @@
 		<!-- Revision variable removes warning about dynamic version -->
 		<revision>${build.version}-SNAPSHOT</revision>
 		<!-- This allows to change between versions and snapshots. -->
-		<build.version>1.4.2</build.version>
+		<build.version>1.4.3</build.version>
 		<build.number>-LOCAL</build.number>
 	</properties>
 
@@ -190,7 +190,7 @@
 		<dependency>
 			<groupId>lv.id.bonne</groupId>
 			<artifactId>dragonfights-v1_19_r1</artifactId>
-			<version>1.4.1</version>
+			<version>1.4.2</version>
 			<scope>compile</scope>
 		</dependency>
 	</dependencies>

